dashboard/test/controllers/level_sources_controller_test.rb

```diff
@@ -171,7 +171,7 @@ def expect_s3_upload
     assert_response :success
     assert_sharing_meta_tags(
```

Comment:
The `assert_sharing_meta_tags` expects a hash with `image_url`, every call to this method was passing `image` so none of those tests were properly checking the image tag